exporters/otlp/testing-internal/src/main/java/io/opentelemetry/exporter/otlp/testing/internal/ManagedChannelTelemetryExporterBuilder.java

```diff
@@ -51,6 +52,10 @@ public TelemetryExporterBuilder<T> setEndpoint(String endpoint) {
     if (!uri.getScheme().equals("https")) {
       channelBuilder.usePlaintext();
     }
+    // User-Agent can only be set at the channel level with upstream gRPC client. If a user wants
+    // the User-Agent to be spec compliant they must manually set the user agent when building
+    // their channel.
```

Comment:
Is this something that should be in docs somewhere? Or javadoc that's more visible than an internal comment?